src/components/Modal/src/Modal.vue

```diff
@@ -1,12 +1,47 @@
 <template>
-	<div :class="$s.Container">
-		<div :class="$s.Modal">
-			<!-- @slot Modal content -->
-			<slot />
+	<div :class="$s.ArrowsContainer">
```

Comment:
where is the `ArrowsContainer` styles? they're not declared in the style tag below, this class is not doing anything